test/unit/test_indent.ml

```diff
@@ -24,11 +24,13 @@ let x =
 y
 in
 2|}
-        ~expected:{|let f =
+        ~expected:
+          (Eol_compat.normalize_eol ~line_endings:`Lf
```

Comment:
Would be better inside the `test` function.

Reply:
done